packages/create-expo-app/src/utils/npm.ts

```diff
@@ -18,15 +18,16 @@ type ExtractProps = {
   cwd: string;
   strip?: number;
   fileList?: string[];
+  disableCache?: boolean;
 };
 
 // @ts-ignore
 const pipeline = promisify(Stream.pipeline);
 
-const cachedFetch = createFetch({
-  cacheDirectory: 'template-cache',
-  // Set no timeout on the templates since they're versioned already.
-});
+function getTemporaryCacheFilePath(subdir: string = 'template-cache') {
+  // This is cleared when the device restarts
+  return path.join(os.tmpdir(), '.create-expo-app', subdir);
```

Comment:
Not sure if relevant, but if we want to support the [XDG base spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), we probably also need to check for `$XDG_CACHE_HOME`

Reply:
Unclear if this applies, since we don't have an example of how XDG overlaps with the tmpdir folder, we should skip. Easy enough to add in the future when we have more clarity.